httpclient/src/main/java/org/apache/http/conn/ssl/SSLConnectionSocketFactory.java

```diff
@@ -183,6 +193,16 @@ public static SSLConnectionSocketFactory getSocketFactory() throws SSLInitializa
         return new SSLConnectionSocketFactory(SSLContexts.createDefault(), getDefaultHostnameVerifier());
     }
 
+    static boolean isWeakCipherSuite(final String cipherSuite) {
+        final String normalizedCipherSuite = cipherSuite.toUpperCase();
```

Comment:
@artem-smotrakov Please use `#toUpperCase` with  `Locale#ROOT` parameter. Alternatively you might want to use non-case sensitive regex matching now.

Reply:
Good point!